### PR TITLE
(portal - analytics / monitoring): Fixing a minor style issue on app. dashboard page

### PR DESCRIPTION
**Pull request by ishangupta-ds**
## Proposed changes

* fixing on hover and focused styles for dropdown

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

File: litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts [hunk 1]
```diff
@@ -96,6 +96,21 @@ const useStyles = makeStyles((theme) => ({
     background: theme.palette.background.paper,
     borderRadius: 4,
     textTransform: 'none',
+    '&:hover': {
+      borderColor: theme.palette.primary.main,
+      boxShadow: `0px 4px 5px -2px ${fade(
+        theme.palette.highlight,
```
Comment (whitetiger1399) [inline litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts:102]: should the boxshadow be -2px here, May you please review it with figma

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts:102]: yes its the same on figma and litmus-ui button too (it's required for transition here) @whitetiger1399

File: litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts [hunk 2]
```diff
@@ -96,6 +96,21 @@ const useStyles = makeStyles((theme) => ({
...
+      )},0px 7px 10px 1px ${fade(
+        theme.palette.highlight,
+        0.14
+      )},0px 2px 16px 1px ${fade(theme.palette.highlight, 0.12)}`,
+      background: theme.palette.background.paper,
+    },
+    '&$focused': {
+      borderColor: theme.palette.primary.main,
+      background: theme.palette.background.paper,
+    },
```
Comment (whitetiger1399) [inline litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts:113]: Can property background removed from focused as its same as parent

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts:113]: On removing it the style is getting overridden explicitly to card header colour and then the background and drop down focused background become of same colour but adding this explicitly solved the issue @whitetiger1399 we need the explicit override here I have added at both places where error was happening

Comment (whitetiger1399) [reply litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts:113]: Ok

**Review by whitetiger1399 — APPROVED** [review]

**Pull request merged**